lib/routes/account.js

```diff
@@ -569,7 +570,9 @@ module.exports = function (
                     })
                   }
 
-                  hasVerifiedSessionFromIp = verified
+                  if (ipProfilingEnabled) {
+                    hasVerifiedSessionFromIp = verified
```

Comment:
@rfk and I were talking about this a little last week, and to be conservative, it may make sense to start with when the recency is a `day`. The charts may have changed in a week, I haven't looked, but last week the huge majority of logs showed the recency was `day`. So starting there has the biggest impact, and we can worry about expiring later, since an IP likely hasn't changed in a day.